src/SDK/Language/PHP.php

```diff
@@ -384,7 +384,10 @@ public function getFilters(): array
         return [
             new TwigFilter('getReturn', function ($value) {
                 return $this->getReturn($value);
-            })
+            }),
+            new TwigFilter('deviceInfo', function ($value) {
+                return php_uname('s') . ';' . php_uname('v') . ';' . php_uname('m');
```

Comment:
For PHP, every value is only separated by `;` and no spaces. This isn't consistent between the various user agents.

Reply:
this part is just os name, type and arch separated by `;` same as other SDKs?

Reply:
The other SDKs have it separated by `; ` (extra space) rather than just `;`